WordPress/gutenberg
# [RNMobile] Move MediaUploadPorgress to its own component folder

`gutenberg-mobile` PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/1361

## Description
This small PR moves `MediaUploadProgress` out of `image` (native) to its own component folder. In favor of https://github.com/wordpress-mobile/gutenberg-mobile/issues/1313 to be able to reuse this for the Media & Text block.

## How has this been tested?
- Add an image block and check it works as expected: upload bar, placeholder.
- Add a video block and check it works as expected: upload bar, placeholder.

## Screenshots
N / A

## Types of changes
Refactor (moves files to another directory)

## Checklist:
